assets/js/common/Table/search.js

```diff
@@ -0,0 +1,23 @@
+const canonicalise = (val) => val.normalize().trim().toLowerCase();
```

Comment:
It's a bit difficult as `normalize()` is already taken :stuck_out_tongue: but maybe something like:

```suggestion
const regularize = (str) => str.normalize().trim().toLowerCase();
```

Reply:
I agree. I like the word canonical, but don't mind changing it, since it's essentially the same thing :duck: